src/jit/jitconfigvalues.h

```diff
@@ -218,6 +218,10 @@ CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 0)
 CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 1)
 #endif // !defined(DEBUG) && !defined(_DEBUG)
 
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+CONFIG_INTEGER(JitMinOptsTrackGCrefs, W("JitMinOptsTrackGCrefs"), 0) // Allow GC roots to be tracked w/MinOpts
```

Comment:
Hmm, hold on - are you suggesting we ditch all the #ifdef's as well? That's not as obvious because even if it's normally disabled, it adds a bunch of code (mostly only 'if' statements but still) but more importantly in the GC decoder it weakens an assert that I think is quite important (I wanted to add an explicit bit so we don't have to assume we must have the new behavior but there's not obvious room for a new flag plus nobody else though it was worth a flag anyway). The suggestion to keep it out of the non-CORECLR build came from the VM folks so how about I leave it the way it is, since it's only been tested that way, and once it's proven its worth (which has yet to be verified independently) it can be enabled in more places?